methods/linkedList/linkedList.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const algolib = require('../index.js');
+
+class LinkedList {
+
+    constructor(){
+        this._elements = [];
+        this.last = -1;
+    }
+
+    push(value){
+        this.last++;
+        let obj = {};
+
+        obj.value = value;
+        this._elements[this.last] = obj;
+
+        if (this.last > 0){
+           obj = this._elements[this.last - 1];
```

Comment:
it would be better to assign new variable instead of "obj" because this one is used in previous operations. This is needed to avoid misunderstanding in reading of code.